btsieve/src/bitcoin/blockchain_info_hex_bitcoin_http_blocksource.rs

```diff
@@ -0,0 +1,169 @@
+use crate::blocksource::{self, BlockSource};
+use bitcoin_support::{deserialize, MinedBlock, Network};
+use futures::{Future, Stream};
+use reqwest::r#async::Client;
+use serde::Deserialize;
+use std::time::Duration;
+use tokio::timer::Interval;
+
+#[derive(Deserialize)]
+struct BlockchainInfoLatestBlock {
+    hash: String,
+    height: u32,
+}
+
+#[derive(Debug)]
+pub enum Error {
+    UnsupportedNetwork(String),
+    Reqwest(reqwest::Error),
+    Hex(hex::FromHexError),
+    BlockDeserialization(String),
+}
+
+#[derive(Clone)]
+pub struct BlockchainInfoHexHttpBlockSource {
+    network: Network,
+    client: Client,
+}
+
+impl BlockchainInfoHexHttpBlockSource {
+    pub fn new(network: Network) -> Result<Self, Error> {
+        // Currently configured for Mainnet only because no support for hex-encoded
+
+        match network {
+            Network::Mainnet => Ok(Self {
+                network,
+                client: Client::new(),
+            }),
+            _ => {
+                log::error!(
+                    "Network {} not supported for bitcoin http blocksource",
+                    network
+                );
+                Err(Error::UnsupportedNetwork(format!(
+                    "Network {} currently not supported for bitcoin http plocksource",
```

Comment:
```suggestion
                     "Network {} not supported by BlockchainInfo (Bitcoin)",
```

Reply:
I cleanup up the comments, now there is only one in the constructor that explains why. Hope that is more understandable :)